WzComparerR2/CharaSimControl/SetItemTooltipRender.cs

```diff
@@ -342,7 +342,7 @@ private void RenderEffect(Graphics g, bool specialPetSetEffectName, ref int picH
                         List<Potential> ops = (List<Potential>)prop.Value;
                         foreach (Potential p in ops)
                         {
-                            GearGraphics.DrawPlainText(g, p.ConvertSummary(), GearGraphics.EquipDetailFont2, color, 10, 244, ref picHeight, 15);
+                            GearGraphics.DrawString(g, p.ConvertSummary(), GearGraphics.EquipDetailFont2, 10, 244, ref picHeight, 15, null, color);
```

Comment:
```suggestion
                            GearGraphics.DrawString(g, p.ConvertSummary(), GearGraphics.EquipDetailFont2, 10, 244, ref picHeight, 15, textColor: color);
```

https://docs.microsoft.com/en-us/dotnet/csharp/methods#optional-parameters-and-arguments 이 형태가 괜찮아 보입니다.